src/RecognitionManager.ts

```diff
@@ -1,19 +1,31 @@
-import { isNative } from "./NativeSpeechRecognition";
-import isAndroid from "./isAndroid";
-import { browserSupportsPolyfills, concatTranscripts, debounce } from "./utils";
+import { debounce } from "lodash";
```

Comment:
I previously inlined `debounce` to avoid importing `lodash` and bloating the library. If there's an option to selectively import only `lodash.debounce`, I would prefer this. Otherwise, inlining this simple function like before keeps the library dependencies to a minimum.

Reply:
Yep, that's what I just thought about!